apps/vs-code-designer-react/src/app/app.tsx

```diff
@@ -30,9 +34,21 @@ export const App = () => {
     isMonitoringView,
     runId,
   } = vscodeState;
-  const [standardApp, setStandardApp] = useState(panelMetaData?.standardApp);
+  const [standardApp, setStandardApp] = useState<StandardApp | undefined>(panelMetaData?.standardApp);
   const [runInstance, setRunInstance] = useState<LogicAppsV2.RunInstanceDefinition | null>(null);
   const [theme, setTheme] = useState<Theme>(getTheme(document.body));
+  const intl = useIntl();
+
+  const intlText = {
```

Comment:
Hi @ecfan these are the only text I added, both of them have screenshots in the PR description

Reply:
@ccastrotrejo: I noticed the following text, which shouldn't use the word "please" [except in specific cases](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please):

![image](https://user-images.githubusercontent.com/13206671/222923703-8d6dff08-eecf-4e78-a932-220587d2f608.png)

Reply:
@ecfan That isn't our text, that is the text that Apple includes on the terminal to let you know it's ZSH, separate from us entirely

Reply:
👍🏻😢